index.html

```diff
@@ -72,7 +72,7 @@ <h2 class="connect-btn1 desktop-connect-btn">let's connect</h2>
         </section>
     </header>
 
-    <main >
+    <main>
         <!-- First page -->
         <!-- work Page -->
         <section id="work-section" class="work-section">
```

Comment:
- [ ] The projects section is created dynamically with the information from the projects array (all the HTML of the section is created when the page loads). This requires changes in existing code - already created HTML code for projects list should be removed, and instead it should be added via JavaScript. so kindly remove all of that from HTML and add it via javascript file ✅✅